armi/tests/test_runLog.py

```diff
@@ -285,8 +285,10 @@ def test_concatenateLogs(self):
             runLog.concatenateLogs(logDir=logDir)
 
             # verify output
-            combinedLogFile = os.path.join(logDir, "armi-mpi-workers.log")
-            self.assertTrue(os.path.exists(combinedLogFile))
+            #logFile = glob(os.path.join(logDir, "*-mpi.log"))
```

Comment:
Can we remove all unused (commented out code) from this PR?

(Though I LOVE that the unit tests were touched!)